### DISCUSSION TITLE
挂载阿里云盘，提示token有3部分，这个怎么处理呀？

### DISCUSSION THREAD

**Discussion by princenjj** [AlistGo/alist#4286] [Q&A]
挂载阿里云盘提示
failed get obj: failed to list objs: failed to refresh token: The token was expected to have 3 parts, but got 1.

救救孩子吧😭😭

**Comment by gaoyuan5232** [comment]
我和你一样 目前有解决吗？

**Comment by Ouyangchenxing** [comment]
我也这吊样。。。挂不上

**Comment by dreamsamethyst** [comment]
俺也一样

**Reply by gaoyuan5232** [reply]
各位，我解决了，整个删了重新装一遍 解决
